### DISCUSSION TITLE
Transfer的右半部功能栏，能否也是tree？

### DISCUSSION THREAD

**Discussion by pxim** [ant-design/ant-design#37276] [General]
![image](https://user-images.githubusercontent.com/5111701/186603610-4de0be4c-eb6d-45ff-8db7-346dfdafd44c.png)
![image](https://user-images.githubusercontent.com/5111701/186603731-0e6a23cc-3d15-45e9-91eb-9f1c6408ebb9.png)

Transfer的右半部功能栏，能否也是tree，类似图2的拖动tree，我试了，没解决。

### What problem does this feature solve?

解决选项层级问题。

### What does the proposed API look like?

。

**Comment by pxim** [comment]
![image](https://user-images.githubusercontent.com/5111701/186812997-c8059d0e-d696-42cc-abaa-45ee4e49e20f.png)
![image](https://user-images.githubusercontent.com/5111701/186813075-8c3d2c31-8803-4c35-be2d-6e17c3eeb7d5.png)
我在 右半边框，也用Tree的话，目前 Transfer 单项模式，右边框item的remove按钮没了，我该如何自定义实现这个功能，没找到相关 remove事件接口
@zombieJ @chenshuai2144

**Reply by MichaelWhite98** [reply]
想问一下你的图标是怎么加的。

**Comment by wangxing0204** [comment]
解决了?目前我在试着弄右边的tree

**Reply by pxim** [reply]
没解决，左右树得数据同步，各自层级 这个问题没解决

**Reply by wangxing0204** [reply]
网上也没找到

**Comment by resetsix** [comment]
写了一个demo
https://github.com/resetsix/Antd_Tree-to-Tree_Transfer
